corehq/apps/importer/tasks.py

```diff
@@ -64,13 +64,22 @@ def do_import(spreadsheet_or_error, config, domain, task=None, chunksize=CASEBLO
     ids_seen = set()
 
     def _submit_caseblocks(caseblocks):
+        err = False
         if caseblocks:
-            submit_case_blocks(
-                [ElementTree.tostring(cb.as_xml()) for cb in caseblocks],
-                domain,
-                username,
-                user_id,
-            )
+            try:
+                submit_case_blocks(
+                    [ElementTree.tostring(cb.as_xml()) for cb in caseblocks],
+                    domain,
+                    username,
+                    user_id,
+                )
+            except Exception:
```

Comment:
There is also a class of errors that won't raise an exception but can be found by examining the form:
```python
response, xform, cases = submit_form_locally(...)
if not xform.is_error:
   problem = xform.problem
```

Sorry I should have mentioned that before